semanticdb/semanticdb3/guide.md

```diff
@@ -0,0 +1,576 @@
+# SemanticDB Guide
+
+SemanticDB is a data model for semantic information about programs in Scala and
+other languages. SemanticDB decouples production and consumption of semantic
+information, establishing documented means for communication between tools.
+
+In this document, we introduce practical aspects of working with SemanticDB.
+We describe the tools that can be used to produce SemanticDB payloads, the tools
+can be used to consume SemanticDB payloads and useful tips & tricks for working
+with SemanticDB. If you're looking for a comprehensive reference of SemanticDB
+features, check out [the specification](semanticdb3.md).
+
+* [SemanticDB example](#semanticdb-example)
+* [What is SemanticDB good for?](#what-is-semanticdb-good-for)
+* [Producing SemanticDB](#producing-semanticdb)
+  * [Scalac compiler plugin](#scalac-compiler-plugin)
+  * [Metac](#metac)
+  * [Sbt](#sbt)
+  * [Metacp](#metacp)
+* [Consuming SemanticDB](#consuming-semanticdb)
+  * [Scalameta](#scalameta)
+  * [Protoc](#protoc)
+  * [Metap](#metap)
+
+## SemanticDB example
```

Comment:
I propose we have a dedicated "Installation" section above here listing all the tools with a single sentence explanation and coursier alias. It's currently not possible to follow along this example since it uses `metac`/`metap` that the user doesn't know how to install.

Down the road, I'd like to provide a `brew install scalameta/scalameta` that installs everything in one command and handles coursier/native/etc